tests/11-basic-basic_errors.c

```diff
@@ -175,6 +175,39 @@ int main(int argc, char *argv[])
 	seccomp_release(ctx);
 	ctx = NULL;
 
+	/* seccomp_export_bpf_buf errors */
+	char buf[1024];
+	size_t buf_len = sizeof(buf);
+	rc = seccomp_export_bpf_buf(ctx, buf, &buf_len);
+	if (rc != -EINVAL)
+		return -1;
+
+	ctx = seccomp_init(SCMP_ACT_KILL);
+	if (ctx == NULL)
+		return -1;
+	rc = seccomp_export_bpf_buf(ctx, buf, NULL);
+	if (rc != -EINVAL)
+		return -1;
+	rc = seccomp_export_bpf_buf(ctx, NULL, NULL);
+	if (rc != -EINVAL)
+		return -1;
+
+	rc = seccomp_export_bpf_buf(ctx, NULL, &buf_len);
```

Comment:
nitpick - you've overwritten buf_len here.  Do you want to reset it back to the size of the buf after this call?  Or is the following test code now using/expecting this smaller buf value to handle the case when the program is larger than the buffer?  If so, I think we will want to add a comment here.

Reply:
It feels like the latter to me.

Reply:
done